### PR TITLE
Release cluster agent 7.20.1

### PR DESCRIPTION
**Pull request by zippolyte**
### What does this PR do?

Initial release of the cluster agent bosh release package, containing the cluster agent 7.20.1

### Review checklist (to be filled by reviewers)

- [ ] PR title must be written as a CHANGELOG entry [(see why)](/CONTRIBUTING.md#pull-request-title)
- [ ] Files changes must correspond to the primary purpose of the PR as described in the title (small unrelated changes should have their own PR)
- [ ] PR must have one `changelog/` label attached. If applicable it should have the `backward-incompatible` label attached.
- [ ] PR should not have `do-not-merge/` label attached.
- [ ] If Applicable, issue must have `kind/` and `severity/` labels attached at least.

### CODE REVIEW

File: config/blobs.yml [hunk 1]
```diff
@@ -1 +1,4 @@
---- {}
+datadog-cluster-agent/datadog-cluster-agent-cloudfoundry.tar.xz:
```
Comment (gzussa) [inline config/blobs.yml:1]: `xz`?

File: scripts/prepare.sh [hunk 2]
```diff
@@ -3,9 +3,9 @@ set -e
 
 BLOBS_D="$(pwd)/blobs"
 BLOB="datadog-cluster-agent/datadog-cluster-agent-cloudfoundry.tar.xz"
-DEFAULT_CLUSTER_AGENT_VERSION="7.20.0-rc.2.git.2.68e6468"
+DEFAULT_CLUSTER_AGENT_VERSION="7.20.1"
 CLUSTER_AGENT_VERSION=${1:-$DEFAULT_CLUSTER_AGENT_VERSION}
-DOWNLOAD_URL="http://s3.amazonaws.com/dsd6-staging/linux/cluster-agent-cloudfoundry/datadog-cluster-agent-cloudfoundry-${CLUSTER_AGENT_VERSION}-amd64.tar.xz"
+DOWNLOAD_URL="http://s3.amazonaws.com/dd-agent/agent7/cluster-agent-cloudfoundry/linux/datadog-cluster-agent-cloudfoundry-${CLUSTER_AGENT_VERSION}-amd64.tar.xz"
```
Comment (gzussa) [inline scripts/prepare.sh:8]: Why is this different?

File: scripts/publish.sh [hunk 3]
```diff
@@ -20,6 +20,7 @@ fi
 
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 WORKING_DIR="$DIR/.."
+RELEASE_ARCHIVE_NAME="datadog-cluster-agent-boshrelease"
```
Comment (gzussa) [inline scripts/publish.sh:23]: is the value correct? `datadog-agent-cluster-release` or `datadog-cluster-agent-boshrelease`

File: scripts/prepare.sh (hunk 2 above)
Comment (zippolyte) [reply scripts/prepare.sh:8]: We use the prod repo now

File: config/blobs.yml (hunk 1 above)
Comment (zippolyte) [reply config/blobs.yml:1]: it's just the compression format.

File: scripts/publish.sh (hunk 3 above)
Comment (zippolyte) [reply scripts/publish.sh:23]: the later, for consistency with the `datadog-agent-boshrelease`

**Pull request merged**